module/Decision/src/Decision/Controller/AdminController.php

```diff
@@ -66,6 +66,42 @@ public function deleteDocumentAction()
         return $this->redirect()->toRoute('admin_decision/document');
     }
 
+    public function changePositionDocumentAction()
```

Comment:
Do I need to check permissions here?

Reply:
Yes, you should. I think you can use something like `$this->isAllowed('delete_document', 'meeting')`